src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -1392,9 +1393,18 @@ public string EffectivePath
 
             public Stream GetStream()
             {
+                _lastWriteTime = DateTimeOffset.UtcNow;
```

Comment:
can this be set when the field is declared? It doesn't seem like this needs to happen here

Reply:
@eerhardt updated to initialize it at declaration point